server/config.go

```diff
@@ -385,7 +385,7 @@ const (
 	defaultMaxReplicas          = 3
 	defaultMaxSnapshotCount     = 3
 	defaultMaxPendingPeerCount  = 16
-	defaultMaxMergeRegionSize   = 0
+	defaultMaxMergeRegionSize   = 20
```

Comment:
20 byte?

Reply:
20MB.

Reply:
can you make it more specifical?

Reply:
@siddontang what do you mean? declare unit explicitly

Reply:
use 20 * 1024 * 1024

or rename to RegionSizeMb exjplictily. 

IMO, size is for byte, not mega byte.

Reply:
I prefer renaming to RegionSizeMB. Size for byte is useless cause the unit of region size in pd is MB.